src/librustc_mir/dataflow/mod.rs

```diff
@@ -590,8 +590,8 @@ impl<E:Idx> GenKillSet<E> {
         self.kill_set.remove(e);
     }
     fn gen_all<I>(&mut self, i: I)
```

Comment:
```suggestion
    fn gen_all(&mut self, i: impl IntoIterator<Item: Borrow<E>>)
```
(might not work due to inference, but worth a try)